src/Containers.jsx

```diff
@@ -202,13 +202,17 @@ class Containers extends React.Component {
         const columnTitles = [_("Name"), _("Image"), _("Command"), _("CPU"), _("Memory"), _("State")];
 
         let emptyCaption = _("No containers");
-        if (this.props.textFilter.length > 0)
+        if (this.props.containers === null)
+            emptyCaption = _("Loading containers...");
```

Comment:
As the tiniest nitpick: Could we just have "Loading..." to make translation easier, and be more flexible in the future about other spinners without breaking i18n?